pulsar-client/src/main/java/org/apache/pulsar/client/impl/TableViewImpl.java

```diff
@@ -127,12 +128,12 @@ public boolean containsKey(String key) {
 
     @Override
     public T get(String key) {
-       return data.get(key);
+        return data.get(key);
```

Comment:
Please undo formattting

Reply:
Done, thanks @eolivelli and @gaozhangmin! My IntelliJ did the reformatting automatically - is there a code formatter that this repository uses?